app/main.py

```diff
@@ -14,4 +14,5 @@ def count_occurrences(phrase: str, letter: str) -> int:
     :param letter: letter to find occurrences of it
     :return: count occurrences of letter in phrase
     """
-    # write your code here
+
+    return sum(map(lambda x: 1 if x == letter.lower() else 0, phrase.lower()))
```

Comment:
As I see you know some advanced features, but now your code is overcomplicated. I bet you can quickly come up with an easier solution which would be in a more "pythonic way"